exporters/committime/collector_base.py

```diff
@@ -251,6 +306,78 @@ def get_repo_from_jenkins(self, jenkins_builds):
                 )
         # If no repo is found, we will return None, which will be handled later on
 
+    def get_metrics_from_pipelineruns(
```

Comment:
Should this be a "private" method ? _get_metrics_from_pipelineruns()

Reply:
Good call!